indexer/e2e/verify-indexer-liquidity.test.ts

```diff
@@ -0,0 +1,523 @@
+import pg from 'pg';
```

Comment:
Again, those are not e2e test. DB is not exposed via the indexer. GraphQL query expose the data stored by indexer.
We either rework it to be true e2e tests or just name those integration tests.
I'll rather have e2e test because it give us more confidence in the code actually working.

Reply:
There are both, DB and API tests.